### PR TITLE
module 12 attempt 2 + module 13

### PR DESCRIPTION
**Pull request by red031000**

### CODE REVIEW

File: arm9/modules/12/asm/module_12.s [hunk 1]
```diff
@@ -48331,65 +48428,134 @@ MOD12_02245074: ; 0x02245074
...
+	.byte 0x24, 0x00, 0x26, 0x00, 0x27, 0x00, 0x00, 0x00
+
+	.global MOD12_022451F4
+MOD12_022451F4: ; 0x022451F4
+	.byte 0x58, 0x00, 0x0C, 0x00, 0x78, 0x00, 0x0C, 0x00
+	.byte 0x98, 0x00, 0x0C, 0x00
+
+	.global MOD12_02245200
+MOD12_02245200: ; 0x02245200
+	.byte 0x60, 0x00, 0x14, 0x00, 0x80, 0x00, 0x14, 0x00, 0xA0, 0x00, 0x14, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/12/asm/module_12.s:48482]: Type: `s16[][2]` (`ldrsh`)

File: arm9/modules/12/asm/module_12.s [hunk 2]
```diff
@@ -48331,65 +48428,134 @@ MOD12_02245074: ; 0x02245074
...
-	.byte 0x98, 0x00, 0x0C, 0x00, 0x60, 0x00, 0x14, 0x00, 0x80, 0x00, 0x14, 0x00, 0xA0, 0x00, 0x14, 0x00
-	.byte 0x02, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x05, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
-	.byte 0x28, 0x00, 0x00, 0x00, 0x46, 0x00, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00, 0x05, 0x00, 0x2B, 0x00
-	.byte 0x05, 0x00, 0x5B, 0x00, 0x05, 0x00, 0x8B, 0x00, 0x05, 0x00, 0xBB, 0x00, 0x00, 0x00, 0x00, 0x00
+	.byte 0x24, 0x00, 0x26, 0x00, 0x27, 0x00, 0x00, 0x00
+
+	.global MOD12_022451F4
+MOD12_022451F4: ; 0x022451F4
+	.byte 0x58, 0x00, 0x0C, 0x00, 0x78, 0x00, 0x0C, 0x00
+	.byte 0x98, 0x00, 0x0C, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/12/asm/module_12.s:48478]: Type: `s16[][2]`

File: arm9/modules/12/asm/module_12.s [hunk 3]
```diff
@@ -48331,65 +48428,134 @@ MOD12_02245074: ; 0x02245074
...
+	.global MOD12_022451D4
+MOD12_022451D4: ; 0x022451D4
+	.byte 0x12, 0x00, 0x13, 0x00, 0x14, 0x00, 0x15, 0x00
 	.byte 0x16, 0x00, 0x0C, 0x00, 0x0D, 0x00, 0x0E, 0x00, 0x0F, 0x00, 0x10, 0x00, 0x29, 0x00, 0x28, 0x00
-	.byte 0x24, 0x00, 0x26, 0x00, 0x27, 0x00, 0x00, 0x00, 0x58, 0x00, 0x0C, 0x00, 0x78, 0x00, 0x0C, 0x00
-	.byte 0x98, 0x00, 0x0C, 0x00, 0x60, 0x00, 0x14, 0x00, 0x80, 0x00, 0x14, 0x00, 0xA0, 0x00, 0x14, 0x00
-	.byte 0x02, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x05, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
-	.byte 0x28, 0x00, 0x00, 0x00, 0x46, 0x00, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00, 0x05, 0x00, 0x2B, 0x00
-	.byte 0x05, 0x00, 0x5B, 0x00, 0x05, 0x00, 0x8B, 0x00, 0x05, 0x00, 0xBB, 0x00, 0x00, 0x00, 0x00, 0x00
+	.byte 0x24, 0x00, 0x26, 0x00, 0x27, 0x00, 0x00, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/12/asm/module_12.s:48473]: Type: `u16[]`

File: arm9/modules/12/asm/module_12.s [hunk 4]
```diff
@@ -48331,65 +48428,134 @@ MOD12_02245074: ; 0x02245074
...
+	.byte 0x60, 0x00, 0x14, 0x00, 0x80, 0x00, 0x14, 0x00, 0xA0, 0x00, 0x14, 0x00
+
+	.global MOD12_0224520C
+MOD12_0224520C: ; 0x0224520C
+	.byte 0x02, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x05, 0x00, 0x00, 0x00
+
+	.global MOD12_02245218
+MOD12_02245218: ; 0x02245218
+	.byte 0x00, 0x00, 0x00, 0x00
+	.byte 0x28, 0x00, 0x00, 0x00, 0x46, 0x00, 0x00, 0x00, 0x64, 0x00, 0x00, 0x00
```
Comment (PikalaxALT) [inline arm9/modules/12/asm/module_12.s:48491]: Type: `s32[]`

File: arm9/modules/13/asm/module_13_arm2.s [hunk 5]
```diff
@@ -694,7 +694,7 @@ MOD13_0221EE98: ; 0x0221EE98
 	movne r0, #0
 	ldmneia sp!, {r4, r5, r6, r7, pc}
 	ldr r0, _0221EF5C ; =0x0223F934
```
Comment (PikalaxALT) [inline arm9/modules/13/asm/module_13_arm2.s:696]: Looks like you missed a few.

**Comment by red031000** [comment]
ready for review

**Pull request merged**